smugmug.go

```diff
@@ -224,6 +231,13 @@ func (w *Worker) albumWorker(id int) {
 				log.Debugf("Quitting albumWorker %d", id)
 				return
 			}
+
+			filter := w.cfg.AlbumFilter
+			if len(filter) > 0 && !strings.HasPrefix(strings.ToLower(album.URLPath), strings.ToLower(filter)) { // Improve: use a real 'mathces' function instead of just HasPrefix
```

Comment:
since `filter` is a string, maybe using `if string != ""` is a bit more clear

Reply:
using lowercase here makes the strong assumption that the user wants a case-insensitive filter. Maybe this is the correct thing, maybe not. Using a regexp match can be more powerful here, although a little bit more expensive